src/shared/component/SwaggerURLInput.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import './SwaggerURLInput.scss';
+
+class SwaggerURLInput extends React.Component {
+
+  onSubmit(e) {
```

Comment:
It's a good practice to keep render as the first function of a component (unless there is a constructor)